### PR TITLE
[6646]Edit universe - missing KMS configuration.

### PR DESCRIPTION
**Pull request by nishantSharma459**

### CODE REVIEW

**Comment by nishantSharma459** [comment]
There was an issue while updating the Universe. The KMS config was empty on browser reload. I've added the lifecycle hook in the parent component to take care of that.

**Review by sshev — CHANGES_REQUESTED** [review]
could you pls check my comment

File: managed/ui/src/components/universes/UniverseForm/UniverseForm.js [hunk 1]
```diff
@@ -260,6 +260,12 @@ class UniverseForm extends Component {
     this.props.submitEditUniverseReadReplica(this.getFormPayload(), universeUUID);
   };
 
+  // This hook will help to render the KMS configs on browser reload while editing
+  // the universe. 
+  componentDidMount() {
+    this.props.getKMSConfigs();
```
Comment (sshev) [inline managed/ui/src/components/universes/UniverseForm/UniverseForm.js:266]: `getKMSConfigs()` already exists in `ClusterFields.js` here:
https://github.com/yugabyte/yugabyte-db/blob/3960cc84ee3689c69009cf29792f783ddbae22d2/managed/ui/src/components/universes/UniverseForm/ClusterFields.js#L285-L286
it looks like it should be called unconditionally rather that duplicated in the parent component
could you please check this?

Comment (nishantSharma459) [reply managed/ui/src/components/universes/UniverseForm/UniverseForm.js]: Done.

**Review by sshev — APPROVED** [review]
Looks good 👍 
Change pull request title to `[#6646] Platform: Edit universe - missing KMS configuration` and free to  merge

**Pull request merged**